src/components/checkbox/checkbox.spec.js

```diff
@@ -136,6 +136,19 @@ describe('mdCheckbox', function() {
     expect(checkbox[0]).not.toHaveClass('md-focused');
   });
 
+  it('should skip toggle if scope property is set', function() {
+    var checkbox = compileAndLink('<md-checkbox ng-model="blue"></md-checkbox>');
+    var scope = checkbox.scope();
+
+    scope.skipToggle = true;
```

Comment:
Just to test if the checkbox skips the click, this is needed for the switch. Bust just rethought, it would make more sense to add this to the switch tests, using the switch extending the checkbox.